jose-b64/src/stream/dec.rs

```diff
@@ -69,3 +70,9 @@ impl<T: Update, E: Encoding> Decoder<T, E> {
         Ok(self.next)
     }
 }
+
+impl<T: fmt::Debug, E> fmt::Debug for Decoder<T, E> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Encoder").field("next", &self.next).finish()
```

Comment:
Since you're omitting some fields...

```suggestion
        f.debug_struct("Encoder").field("next", &self.next).finish_non_exhaustive()
```

Reply:
Thanks, I have updated these